### ISSUE TITLE
Issues following the "Getting Started" instructions

### ISSUE THREAD

**Issue by mirisuzanne** [viljamis/vue-design-system#41]
I downloaded the repo, following the [directions provided online](https://github.com/viljamis/vue-design-system/wiki/getting-started). Git clone, npm install, and npm start. I'm on a mac with High Sierra, and Node v8.5.0. Without making any changes to the code, I get these errors:

```
Warning: highlightTheme config option is deprecated. Use the theme property in the editorConfig option instead
```

^ That warning disappears when I remove `styleguide.config.js` line 29 (`highlightTheme: "night"`). I also see:

```
 ERROR  Failed to compile with 1 errors                            15:57:33

 error  in (webpack)-dev-server/client?http://localhost:8080

Module build failed: TypeError: /Users/miriam/Sites/vue-design-system/node_modules/webpack-dev-server/client/index.js: Property object of MemberExpression expected node to be of a type ["Expression"] but instead got null
```

The browser automatically opens to `localhost:8080`, with only the note: `Cannot GET /`. Maybe I missed something implied in the docs?

**Comment by viljamis** [comment]
Hi @mirisuzanne! the highlightTheme warning is caused by another NPM package and I’m hoping it’ll get fixed soon. Let me try to figure out what is causing the other issue and get back to you with details soon.

**Comment by viljamis** [comment]
@mirisuzanne Found the issue, it’s not just you. working on a fix and pushing soon.

**Comment by viljamis** [comment]
@mirisuzanne Happy to tell you that I just released 1.3.2 that should fix the issue you had. 

I will also file a new bug about the highlighTheme warning. That does not currently cause any issues though and you shouldn’t remove the line since that’ll break the code highlighting in the docs.

**Comment by viljamis** [comment]
@mirisuzanne Btw, I created a separate issue for the warning: https://github.com/viljamis/vue-design-system/issues/44

**Comment by mirisuzanne** [comment]
@viljamis wow, so quickly fixed! Thank you!

**Comment by viljamis** [comment]
@mirisuzanne The other issue you were having is now fixed too in pull #48. Will make a new patch release
